.github/workflows/task-build-artifacts.yml

```diff
@@ -56,14 +59,9 @@ jobs:
       - name: Check if node20 is Supported
         id: node20 # TODO: Remove this when node20 is supported on all platforms, or when we drop support for theses platforms
         run: |
-          for platform in ubuntu:bionic centos:7 amazonlinux:2 alpine:3; do
```

Comment:
Why was centos:7 removed?

Reply:
Was removed from our CI [here](https://github.com/RediSearch/RediSearch/pull/4675) (in master - if it still exists in the version branches it will not be removed there).

Reply:
I see it is not supported on any of the version branches as well (which makes sense..)

Reply:
it should be supported on 2.6, but it was listed here just because there was no good reason to remove it (has no effect)

Reply:
Right, 2.6 as well.
It's confusing w.r.t to whether it is supported on latest or not, I prefer it not to appear where it is not supported